R/arg-homogenise.R

```diff
@@ -0,0 +1,219 @@
+#' Standardise a call against formal arguments.
+#'
+#' This is a slower but more thorough version of
```

Comment:
Sure. I think they should be separate because probably no one will be interested in `lang_homogenise()`. It is only useful for tracing dots across the call stack.